### ISSUE TITLE
Vips can (now?) be compiled without webp support - Our class should detect that

### ISSUE THREAD

**Issue by rosell-dk** [rosell-dk/webp-convert#272]

**Comment by rosell-dk** [comment]
I updated the vips version on travis from 8.6.3 to 8.10.6 and now gets the following error:

```
1) WebPConvert\Tests\Convert\Converters\VipsTest::testWebpsave
WebPConvert\Convert\Exceptions\ConversionFailedException: VipsOperation: class "webpsave" not found
/home/travis/build/rosell-dk/webp-convert/src/Convert/Converters/Vips.php:206
/home/travis/build/rosell-dk/webp-convert/tests/BaseExposer.php:43
/home/travis/build/rosell-dk/webp-convert/tests/BaseExposer.php:30
/home/travis/build/rosell-dk/webp-convert/tests/BaseExposer.php:46
/home/travis/build/rosell-dk/webp-convert/tests/Convert/Exposers/VipsExposer.php:37
/home/travis/build/rosell-dk/webp-convert/tests/Convert/Converters/VipsTest.php:190
```

**Comment by rosell-dk** [comment]
Fixed in a4e23e6..8a66dd2
